ignite-generator/src/app/index.es

```diff
@@ -367,7 +367,7 @@ export class AppGenerator extends Generators.Base {
    * Yeoman times its template copies.  :(
    */
   install () {
-    const npmStatus = 'Installing Ignite dependencies (~30 seconds-ish)'
+    const npmStatus = 'Installing Ignite dependencies (~ 1 minute)'
```

Comment:
I don't care, had no idea I loved this until I saw ish, it makes me smile.   I'm sorry but I require it to stay in my life.  Don't make ish and I run off to vegas.